tests/cephadm/test_config.py

```diff
@@ -0,0 +1,60 @@
+import traceback
+
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def set_multiple_public_network(cls):
+    """
+    To set multiple public network to ceph.
+    Args:
+        cls: cephadm instance object
+
+    Returns:
+        False -> Fail, True -> Pass
+    """
+    out, _ = cls.shell(args=["ceph", "config", "get", "mon", "public_network"])
+    ceph_pub_network = out.rstrip()
+    mask = ceph_pub_network.rsplit(sep="/")
+    # logic to create new public ip by incrementing mask count
+    new_mask = int(mask[1]) + 1
+    new_pub_network = mask[0] + "/" + str(new_mask)
+    multiple_network = f"{ceph_pub_network}" + "," + f"{new_pub_network}"
+    cmd = ["ceph", "config", "set", "mon", "public_network", f"{multiple_network}"]
+    try:
+        out, err = cls.shell(args=cmd)
+        log.debug("Set multiple public network to ceph mon successfully")
+    except Exception:
+        log.error("Failed to set multiple public network to ceph")
+        log.error(traceback.format_exc())
+        return False
```

Comment:
Please avoid using general exception. Use exact exception which will be raised by calling function and comprare expected error messages in except if possible